pkg/providers/imagefamily/resolver.go

```diff
@@ -95,6 +109,12 @@ func (r Resolver) Resolve(ctx context.Context, nodeClass *v1alpha2.AKSNodeClass,
 		instancetype.MemoryAvailable: instanceType.Overhead.EvictionThreshold.Memory().String()}
 	kubeletConfig.MaxPods = lo.ToPtr(getMaxPods(staticParameters.NetworkPlugin))
 
+	if staticParameters.NetworkPlugin == networkPluginAzure {
```

Comment:
Does this indeed apply for any network mode? If yes - let's add a comment

Reply:
Removed conditional bit can add back later when I can make a more informed comment